src/components/CioPlpGrid/CioPlpGrid.tsx

```diff
@@ -69,6 +74,12 @@ export default function CioPlpGrid(props: CioPlpGridWithRenderProps) {
     </span>
   );
 
+  if (!searchQuery) {
+    // Render Zero results page in the case where no query is provided
+    // TODO: Allow customers to override default page
+    return <ZeroResults />;
```

Comment:
I think the `ZeroResults` renders something like `We couldn't find "query"`, which would look broken if no string is here. Maybe we can add a validation there to show a different message if there's no `query`

Reply:
Good catch. Updated